src/renderer/Default.js

```diff
@@ -56,7 +56,7 @@ export const TopBannerContainer: ThemedComponent<{}> = styled.div`
 export default function Default() {
   const location = useLocation();
   const ref: React$ElementRef<any> = useRef();
-  const isSwapV2Enabled = true; // useEnv("EXPERIMENTAL_SWAP");
+  const isSwapV2Enabled = useEnv("EXPERIMENTAL_SWAP");
```

Comment:
This means we need to be yalc'ing live-common or is the version that added the flag already merged?

Reply:
Yep, working like a charm without needing to yalc anything.

Reply:
nice